lib/discourse_api/api/topics.rb

```diff
@@ -46,7 +46,7 @@ def change_topic_status(topic_slug, topic_id, params = {})
         params = API.params(params)
           .required(:status, :enabled)
           .optional(:api_username)
-        put("/t/#{topic_slug}/#{topic_id}/status", params.to_h)
+        put("/t/#{topic_id}/status", params.to_h)
```

Comment:
The `topic_slug` parameter on line 45 can also be removed, but removing it is not backwards-compatible so I left it for now.